config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -41,6 +41,56 @@ presubmits:
             memory: "6Gi"
         securityContext:
           privileged: true
+  - name: pull-kubernetes-node-e2e-containerd-decorate
+    skip_branches:
+      - release-\d+\.\d+  # per-release image
+    annotations:
+      fork-per-release: "true"
+      testgrid-alert-stale-results-hours: "24"
+      testgrid-create-test-group: "true"
+      testgrid-num-failures-to-alert: "10"
+      testgrid-dashboards: sig-node-presubmits
+      testgrid-tab-name: pull-kubernetes-node-e2e-containerd-decorate
+    always_run: false
+    optional: true
+    decorate: true
+    path_alias: k8s.io/kubernetes
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+      preset-dind-enabled: "true"
```

Comment:
Why do we need docker-in-docker for this test?

Reply:
I don't think we do.